frontend/public/components/cluster-overview.jsx

```diff
@@ -148,6 +148,9 @@ const LimitedGraphs = ({openshiftFlag}) => {
 };
 
 const GraphsPage = ({fake, limited, namespace, openshiftFlag}) => {
+  if (flagPending(openshiftFlag)) {
+    return null;
```

Comment:
Need to remove the additional (existing) `flagPending(openshiftFlag)` checks in the component below

Reply:
Done